test/test_conversion.py

```diff
@@ -9,9 +9,9 @@
 
 import logging
 
-def test_conversion_adaptive_model(caplog):
```

Comment:
A bit offtopic - but do you think we can remove these captog things from the other tests too?

Reply:
I wouldnt like to, since they suppress the logging output during testing and we log a lot in FARM. With logging enabled it makes it very hard to spot errors in the stack trace.

Or do you have a better solution in mind?

Reply:
I do not have a better solution. I just saw you removing it here and thought it would be clever to do that everywhere. I did not know this logging thing in conjunction with PyTest.